runtime/src/lib.rs

```diff
@@ -675,7 +675,7 @@ pub type Executive = frame_executive::Executive<
 	Block,
 	frame_system::ChainContext<Runtime>,
 	Runtime,
-	AllPalletsReversedWithSystemFirst,
+	AllPalletsWithSystem,
```

Comment:
Is `AllPalletsReversedWithSystemFirst` not available anymore?

Reply:
[It is, but deprecated, and gives a warning if used](https://github.com/paritytech/substrate/blob/d66adfabd7911bf01ab01ec96ec4228307a03e07/frame/support/procedural/src/construct_runtime/mod.rs#L449-L452).
If it should stay then let me know and I'll revert this patch.

Reply:
Ah. No, that's fine then. But can you add this specific change to the PR description? It should not, but it might affect the worker.